crates/iota-move/src/main.rs

```diff
@@ -28,10 +28,10 @@ struct Args {
     pub package_path: Option<PathBuf>,
     /// If true, run the Move bytecode verifier on the bytecode from a
     /// successful build
-    #[arg(long = "path", short = 'p', global = true)]
```

Comment:
Is `short = 'p'` there by default?

Reply:
No, there are no short defaults

Reply:
Then this needs to stay, same below

Reply:
why? this was a bug in the first place, no? `p` is the short for `package_path`, not for `run_bytecode-verifier` nor for `print-diags_to_stderr`.

Reply:
Ah they had the same, all good then I guess